tesseract_task_composer/core/src/task_composer_node.cpp

```diff
@@ -158,6 +163,12 @@ int TaskComposerNode::run(TaskComposerContext& context, OptionalTaskComposerExec
   try
   {
     results = runImpl(context, executor);
+    if (return_value_override_ >= 0)
+    {
+      results.return_value = return_value_override_;
+      results.status_message += " (Return value overridden to " + std::to_string(return_value_override_) + ")";
+      results.color = "yellow";
```

Comment:
Personally, I would vote for a color other than yellow as that is already used elsewhere. Maybe a blue or cyan? I was going to say gray, but that appears to already be used to represent "abort"